src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -703,20 +705,25 @@ def _get_env_vars(self):
             if self.step_input.run_properties.versioning.tag
             else self.step_input.run_properties.versioning.pr_number
         )
+        all_deploy_project_executions: set[ProjectExecution] = next(
+            project_executions
+            for stage, project_executions in self.step_input.run_properties.run_plan.full_plan.items()
```

Comment:
Here is the actual pr namespace linkup fix by using the full plan (aka the stored file) which includes all the other projects within the deploy stage that are running in different processes in the action.